shared/types/base/permission.ts

```diff
@@ -1,15 +1,16 @@
 import { z } from 'zod'
+import { BaseModel } from './common'
 
 export enum PermissionType {
   Answerer = 'answerer',
   Admin = 'admin',
 }
 
-// no BaseModel as Permission table has no id
-export const Permission = z.object({
+// Permission table has no id
```

Comment:
If so.. would it be better to model the Sequelize timestamps as its own object, then have BaseModel inherit from it? this would allow you to avoid the `omit()` declaration

Reply:
done in https://github.com/opengovsg/askgovsg/pull/298/commits/2c73056781865a0ff7a1d03189bc5e85a2839e2e